### PR TITLE
Cert checks

### PR DESCRIPTION
**Pull request by simoheinonen**
Making https://github.com/vyuldashev/flysystem-curlftp/pull/17 configurable

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,8 @@ $adapter = new CurlFtpAdapter([
   'utf8' => true,
   'ssl' => true,
   'timeout' => 90,		// connect timeout
+  'sslVerifyPeer' => 0, // using 0 is insecure, use it only if you know what you're doing
```
Comment (vyuldashev) [inline README.md:37]: why not using `false` instead of `0`?

Comment (simoheinonen) [reply README.md:37]: At least libcurl takes integer https://curl.haxx.se/libcurl/c/CURLOPT_SSL_VERIFYPEER.html

**Comment by simoheinonen** [comment]
@vyuldashev can we get this merged and new version out?

**Comment by vyuldashev** [comment]
Tagged a major release v2.0.0 as it has breaking changes.

**Pull request merged**